ui/yarn.lock

```diff
@@ -14199,7 +14217,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"ember-truth-helpers@npm:^3.0.0":
+"ember-truth-helpers@npm:^3.0.0, ember-truth-helpers@npm:^3.1.1":
```

Comment:
In a lot of places such as this one, running the command to upgrade HDS `yarn up @hashicorp/design-system-components` adds something rather than changing it, despite using `yarn up` instead of `yarn add`.

I'm not sure if this means the new HDS 2.9.0 requires `ember-truth-helpers` from versions 3.0.0 AND 3.1.1, or if `yarn up` just incorrectly added the new version instead of upgrading it. 

Does anyone know which is correct? This shows up in a lot of places in this file as you can see

Reply:
So you would use `yarn add` when adding a new dependency to the project and either use `yarn up` to upgrade to the latest version or specify a different version in the `package.json` file and then do `yarn install` which will automatically update `yarn.lock`.

As far as your question regarding `ember-truth-helpers`, I personally don't pay much attention to the yarn.lock since it has information regarding all dependencies including dependencies of dependencies which isn't necessarily something we need to be aware of unless of course there is an issue with a dependency. It could be that HDS updated a dependency which may have updated their version of `ember-truth-helpers` which is why it is showing up here.

Reply:
Makes sense, thank you!